python/lsst/rubintv/production/mountTorques.py

```diff
@@ -39,6 +39,8 @@
                             ]
 
 AUXTEL_ANGLE_TO_EDGE_OF_FIELD_ARCSEC = 280.0
+MOUNT_IMAGE_WARNING_LEVEL = .25  # this determines the colouring of the cells in the table, yellow for this
```

Comment:
It might be good to have an __all__ to control what gets exported (as it's * in __init__.py).